elk/train.py

```diff
@@ -54,6 +57,7 @@ def train(args):
 
     # save models
     # TODO: use better filename for the pkls, so they don't get overwritten
+    Path(args.trained_models_path).mkdir(parents=True, exist_ok=True)
```

Comment:
nit: `args.trained_models_path` is already a `Path` so we don't have to make it one here